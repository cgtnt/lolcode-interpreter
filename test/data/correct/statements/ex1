HAI

I HAS A c ITZ 5
I HAS A d ITZ 5

VISIBLE c d!

VISIBLE BOTH SAEM c AN d!
VISIBLE BOTH SAEM 0.0 AN 0.0!
VISIBLE SMOOSH "lol" "lmo" "word" MKAY!

KTHXBYE
