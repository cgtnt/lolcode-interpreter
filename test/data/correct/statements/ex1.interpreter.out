55
WIN
WIN
lollmoword
